tools/c7n_gcp/tests/test_query.py

```diff
@@ -34,6 +34,7 @@ def test_gcp_resource_metadata_asset_type():
         'ml-job',
         'ml-model',
         'notebook',
+        'patch-deployment',
```

Comment:
asset inventory supports patch deployment, add the corresponding metadata to the resource type info instead
https://cloud.google.com/asset-inventory/docs/supported-asset-types

Reply:
Fixed. Please check